pkg/drivers/kic/oci/info.go

```diff
@@ -252,9 +252,11 @@ func dockerSystemInfo() (dockerSysInfo, error) {
 	var ds dockerSysInfo
 	rawJSON, err := dockerInfoGetter()
 	if err != nil {
+		klog.Errorf("docker info: %v", err)
```

Comment:
change to warningf to avoid poluting the UI

Reply:
ok